library/geonetwork

```diff
@@ -11,10 +11,18 @@ Tags: 3.0.5-postgres, 3.0-postgres
 GitCommit: e5cf5da76f557481c35b4a4d3e3cac77768a1302
 Directory: 3.0.5/postgres
 
-Tags: 3.2.0, 3.2, latest
+Tags: 3.2.0
 GitCommit: c2af30125f631f6357d3af3837ec4b80f04b5917
 Directory: 3.2.0
```

Comment:
Any particular reason to keep this version listed here?  Removing it here will remove it from the "Supported" section on the Hub readme (and will prevent us from spending cycles rebuilding it on the official build server), but the tag will still be available to users who want it.  (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.)